### DISCUSSION TITLE
How to set the Temperature?

### DISCUSSION THREAD

**Discussion by Algit73** [RageAgainstThePixel/OpenAI-DotNet#221] [Q&A] [answered]
Hi
Thanks for your package.

I just wanted to set the temperature, because it is a must in my project, but I couldn't find it. Would you please tell me if it exists there or should be implemented in another way?

I want it for the chat.

**Comment by StephenHodgson** [comment, marked as answer]
it is a parameter you can set when creating a chat reuest:

```csharp
var request = new ChatRequest(messages, model, temperature: 0.1f);
```
